app/views/articles/_org_branding.html.erb

```diff
@@ -1,6 +1,6 @@
 <% if organization && organization.bg_color_hex.present? %>
   <div class="flex <% if author == true %>mt-6<% end %>">
-    <a href="/<%= organization.slug %>" class="crayons-avatar crayons-avatar--2xl shrink-0 mr-4">
+    <a href="/<%= organization.slug %>" class="crayons-avatar--2xl shrink-0 mr-4">
```

Comment:
Ok, here's the actual fix. It will also remove the circle on top of the logo.

```suggestion
    <a href="/<%= organization.slug %>" class="crayons-logo crayons-logo--2xl shrink-0 mr-4">
```